composer.json

```diff
@@ -1,20 +1,26 @@
 {
-  "name": "mindscape/raygun4wordpress",
```

Comment:
Is it ok to make this change away from mindscape? Doublecheck with product please